src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -32,15 +35,21 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
     },
   ] = useToggle();
   const { user: userByAddress } = useUserByAddress(field.value);
-  const userDisplayName = userByAddress?.profile?.displayName;
+  const { user: userByName } = useUserByName(field.value);
+  const userDisplayName =
+    userByAddress?.profile?.displayName ||
+    userByName?.profile?.displayName ||
+    field.value;
```

Comment:
`field.value` will be a hex address, do we want to show it here? If we do I think we should also check for loading of `useUserByAddress` and `useUserByName` to avoid flashing the hex address while user details are loading